docs-site/content/0.19.0/guide/features/faceting.md

```diff
@@ -1 +1,215 @@
 # Faceting
+
+Faceting is a way to classify results in smaller groups. It helps users to narrow down their search based on certain parameters. For example, in a dataset of books, users may want to search for books by categorizing them first via author, genre, etc. Similarly, in a movie dataset, users may want to search for movies by first grouping them via actor, director, genre, etc. This subclassification of search results is known as Faceting. Faceting is not to be confused with Filtering, as Filtering provides a way to remove results that don't meet certain criteria, while Faceting provides a way to subgroup documents that can be searched upon further.
+
+TypeSense provides out-of-the-box support for Faceting. To facet results, you first need to define a facet field. This field will allow TypeSense to subgroup results into categories and then the search results can be refined in that subgroup. In a book dataset, you can define a facet field on authors or publication year:
+
+<Tabs :tabs="['JavaScript','PHP','Python','Ruby']">
+  <template v-slot:JavaScript>
+
+```javascript
+booksSchema = {
+  'name': 'books',
+  'fields': [
+    {'name': 'title', 'type': 'string' },
+    {'name': 'authors', 'type': 'string[]' },
+    {'name': 'image_url', 'type': 'string' },
+
+    {'name': 'publication_year', 'type': 'int32' },
+    {'name': 'ratings_count', 'type': 'int32' },
```

Comment:
Could you also facet by `ratings_count` and add a note about integer fields returning min, max, avg as well in addition to count.